src/derive.rs

```diff
@@ -283,15 +283,15 @@ pub trait Subcommand: FromArgMatches + Sized {
 /// ```
 pub trait ArgEnum: Sized {
     /// All possible argument choices, in display order.
-    const VARIANTS: &'static [&'static str];
+    fn variants() -> Vec<ArgValue<'static>>;
```

Comment:
Since this isn't all the variants, we should rename.

Ideas
- `arg_values`
- `values`

Reply:
I'll choose `arg_value(s)` for now as I think this prevents possible name clashing and sets clear expectations on what this is.

Reply:
`arg_values` **could** also be possible_values, as that what it is in relation to the `Arg`

Reply:
I see one as the mechanism (what it can do) and the other is a specific policy (how its being used).